File: split/printmeaning.pas
procedure printmeaning;
begin
  printcmdchr(curcmd, curchr);
  if curcmd >= 111 then
  begin
    printchar(58);
    println;
    tokenshow(curchr);
  end
  else if curcmd = 110 then
  begin
    printchar(58);
    println;
    tokenshow(curmark[curchr]);
  end;
end;

File: cpp/printmeaning.cpp
#include "printmeaning.h"
#include "printcmdchr.h"
#include "printchar.h"
#include "println.h"
#include "tokenshow.h"

void printmeaning(void)
{
	printcmdchr(curcmd, curchr);
	if (curcmd >= call)
	{
		printchar(':');
		println();
		tokenshow(curchr);
	}
	else 
		if (curcmd == top_bot_mark)
		{
			printchar(':');
			println();
			tokenshow(curmark[curchr]);
		}
}
